lxd/cluster/config/config.go

```diff
@@ -576,6 +581,14 @@ var ConfigSchema = config.Schema{
 	//  shortdesc: Instance placement scriptlet for automatic instance placement
 	"instances.placement.scriptlet": {Validator: validate.Optional(scriptletLoad.InstancePlacementValidate)},
 
+	// lxdmeta:generate(entities=server; group=miscellaneous; key=instances.migration.stateful)
+	//
+	// ---
+	//  type: bool
+	//  scope: global
+	//  shortdesc: Whether to set `migration.stateful` to `true` for the instance
```

Comment:
We should clarify here that profile and instance level settings of `migration.stateful` override this setting for the relevant instances.

Reply:
```
// lxdmeta:generate(entities=server; group=miscellaneous; key=instances.migration.stateful)
// You can override this setting for relevant instances, either in the instance-specific configuration or through a profile.
// ---
//  type: bool
//  scope: global
//  shortdesc: Whether to set `migration.stateful` to `true` for the instances
```

This isn't actually anything special for this setting, as it's the same for all (? at least most) configurations.
Makes sense to point it out, but this suggestion should be sufficient.

Reply:
fixed